pkg/asset/installconfig/installconfig.go

```diff
@@ -55,7 +57,7 @@ func (a *InstallConfig) Generate(parents asset.Parents) error {
 
 	a.Config = &types.InstallConfig{
 		TypeMeta: metav1.TypeMeta{
-			APIVersion: "v1beta1",
+			APIVersion: "v1beta2",
```

Comment:
I think we want a public constant for "the current version".  For an example, see [here][1] and other changes in that PR.

[1]: https://github.com/openshift/installer/pull/1154/files#diff-02200a3eac2ac223b4117e690cf2d0c7R23